baseplate/server/__init__.py

```diff
@@ -197,3 +202,69 @@ def load_and_run_script():
     config = read_config(args.config_file, server_name=None, app_name=args.app_name)
     configure_logging(config, args.debug)
     args.entrypoint(config.app)
+
+
+def load_and_run_tshell():
+    """Launch a shell for a thrift service."""
+    parser = argparse.ArgumentParser(
+        description="Open a shell with app configuration loaded.",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+
+    parser.add_argument("--debug", action="store_true", default=False,
+        help="enable extra-verbose debug logging")
+    parser.add_argument("--app-name", default="main", metavar="NAME",
+        help="name of app to load from config_file (default: main)")
+    parser.add_argument("config_file", type=argparse.FileType("r"),
+        help="path to a configuration file")
+
+    args = parser.parse_args(sys.argv[1:])
+    config = read_config(args.config_file, server_name=None, app_name=args.app_name)
+    configure_logging(config, args.debug)
```

Comment:
Should I delete this?

Reply:
We probably don't want fileConfig taking over, so it might be best to just manually set up a `logging.basicConfig(level=logging.INFO)` with a simple formatter or something? Not sure how much logging we'd want to see while trying to interact with the REPL.